tfhe/src/shortint/parameters/multi_bit/p_fail_2_minus_64/ks_pbs_gpu.rs

```diff
@@ -75,85 +74,85 @@ pub const PARAM_GPU_MULTI_BIT_GROUP_2_MESSAGE_3_CARRY_3_KS_PBS_GAUSSIAN_2M64:
     message_modulus: MessageModulus(8),
     carry_modulus: CarryModulus(8),
     max_noise_level: MaxNoiseLevel::new(9),
-    log2_p_fail: -64.168,
+    log2_p_fail: -64.367,
     ciphertext_modulus: CiphertextModulus::new_native(),
     encryption_key_choice: EncryptionKeyChoice::Big,
     grouping_factor: LweBskGroupingFactor(2),
     deterministic_execution: false,
 };
 
 // Group 3
-// p-fail = 2^-64.579, algorithmic cost ~ 73, 2-norm = 3
+
+// p-fail = 2^-64.624, algorithmic cost ~ 60, 2-norm = 3
 pub const PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_1_CARRY_1_KS_PBS_GAUSSIAN_2M64:
     MultiBitPBSParameters = MultiBitPBSParameters {
-    lwe_dimension: LweDimension(786),
-    glwe_dimension: GlweDimension(4),
-    polynomial_size: PolynomialSize(512),
+    lwe_dimension: LweDimension(747),
+    glwe_dimension: GlweDimension(2),
+    polynomial_size: PolynomialSize(1024),
     lwe_noise_distribution: DynamicDistribution::new_gaussian_from_std_dev(StandardDev(
-        8.135492209468212e-06,
+        1.5944604865450687e-05,
     )),
     glwe_noise_distribution: DynamicDistribution::new_gaussian_from_std_dev(StandardDev(
         2.845267479601915e-15,
     )),
-    pbs_base_log: DecompositionBaseLog(22),
+    pbs_base_log: DecompositionBaseLog(21),
     pbs_level: DecompositionLevelCount(1),
     ks_base_log: DecompositionBaseLog(4),
     ks_level: DecompositionLevelCount(3),
     message_modulus: MessageModulus(2),
     carry_modulus: CarryModulus(2),
     max_noise_level: MaxNoiseLevel::new(3),
-    log2_p_fail: -64.579,
+    log2_p_fail: -64.624,
     ciphertext_modulus: CiphertextModulus::new_native(),
     encryption_key_choice: EncryptionKeyChoice::Big,
     grouping_factor: LweBskGroupingFactor(3),
     deterministic_execution: false,
 };
-// p-fail = 2^-65.005, algorithmic cost ~ 79, 2-norm = 5
+// p-fail = 2^-64.256, algorithmic cost ~ 78, 2-norm = 5
 pub const PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_2_CARRY_2_KS_PBS_GAUSSIAN_2M64:
     MultiBitPBSParameters = MultiBitPBSParameters {
-    lwe_dimension: LweDimension(843),
+    lwe_dimension: LweDimension(909),
     glwe_dimension: GlweDimension(1),
     polynomial_size: PolynomialSize(2048),
     lwe_noise_distribution: DynamicDistribution::new_gaussian_from_std_dev(StandardDev(
-        3.0428360625436423e-06,
+        9.743962418842052e-07,
     )),
     glwe_noise_distribution: DynamicDistribution::new_gaussian_from_std_dev(StandardDev(
         2.845267479601915e-15,
     )),
     pbs_base_log: DecompositionBaseLog(22),
     pbs_level: DecompositionLevelCount(1),
-    ks_base_log: DecompositionBaseLog(3),
-    ks_level: DecompositionLevelCount(5),
+    ks_base_log: DecompositionBaseLog(5),
+    ks_level: DecompositionLevelCount(3),
```

Comment:
Did you use the GPU cost model to generate the parameters? It seems strange that the KS levels are reduced and n increases :thinking:

Reply:
Btw I'm running the multi-bit fast benches to check the effect on performance.

Reply:
It worsens performance a bit

Reply:
cc @samcrx

Reply:
Yes I've used it. The one with the values for 256bits chunks